src/test/java/net/sf/jabref/importer/fileformat/MsBibImporterTest.java

```diff
@@ -0,0 +1,102 @@
+package net.sf.jabref.importer.fileformat;
+
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import net.sf.jabref.Globals;
+import net.sf.jabref.JabRefPreferences;
+import net.sf.jabref.bibtex.BibtexEntryAssert;
+import net.sf.jabref.importer.OutputPrinterToNull;
+import net.sf.jabref.model.entry.BibEntry;
+
+/**
+ *
+ * @author tschechlovdev, mairdl
+ *
+ */
+public class MsBibImporterTest {
+
+    @Before
+    public void setUp() throws Exception {
+        if (Globals.prefs == null) {
+            Globals.prefs = JabRefPreferences.getInstance();
+        }
+    }
+
+    @Test
+    public final void testIsRecognizedFormat() throws Exception {
+        MsBibImporter testImporter = new MsBibImporter();
+        //test with files, which have non recognized format
+        List<String> notAccepted = Arrays.asList("CopacImporterTest1.txt", "IsiImporterTest1.isi",
+                "IsiImporterTestInspec.isi", "emptyFile.xml", "IsiImporterTestWOS.isi");
+        for (String s : notAccepted) {
+            try (InputStream stream = MsBibImporter.class.getResourceAsStream(s)) {
+                Assert.assertFalse(testImporter.isRecognizedFormat(stream));
+            }
+        }
+
+        //test with recognized format files
+        List<String> Accepted = Arrays.asList("MsBibImporterTest1.xml", "MsBibImporterTest.xml");
+        for (String s : Accepted) {
+            try (InputStream stream = MsBibImporter.class.getResourceAsStream(s)) {
+                Assert.assertTrue(testImporter.isRecognizedFormat(stream));
+            }
+        }
+
+    }
+
+    @Test
+    public final void testImportEntries() throws java.io.IOException {
+        MsBibImporter testImporter = new MsBibImporter();
+
+        //test with the normal xml File, that was exported by Jabref
+        List<BibEntry> entries = testImporter.importEntries(
+                MsBibImporterTest.class.getResourceAsStream("MsBibImporterTest1.xml"), new OutputPrinterToNull());
+        BibEntry entry = entries.get(0);
+        Assert.assertEquals(12, entry.getFieldNames().size());
+        Assert.assertEquals(1, entries.size());
+
+        //Test with xml file, that has only one child named source
+        entries = testImporter.importEntries(MsBibImporterTest.class.getResourceAsStream("MsBibImporterTest.xml"),
```

Comment:
Better use a string here instead of a file. (Have a look at the other pull request how to do this)